.github/workflows/post_release_version_bump.yml

```diff
@@ -0,0 +1,52 @@
+name: Post-Release Version Bump
+
+on:
+  release:
+    types:
+      - published
+
+jobs:
+  post_release_version_bump:
+    runs-on: ubuntu-latest
+
+    steps:
+      - uses: actions/checkout@v2
+
+      - name: Derive versions
+        id: versions
+        run: |
+          old_version=`grep -oP '(?<=__version__ = ").*(?=")' xir/_version.py`
+          new_version=`echo $old_version | awk '{split($0,v,"."); print v[1] "." v[2]+1 "." v[3] "-dev"}'`
+
+          echo "::set-output name=old_version::$old_version"
+          echo "::set-output name=new_version::$new_version"
+
+      - name: Bump version in _version.py
+        run:
+          sed -i 's/${{ steps.versions.outputs.old_version }}/${{ steps.versions.outputs.new_version }}/' xir/_version.py
+
+      - name: Bump version in changelog
+        run: |
+          next_release_version=${{ steps.versions.outputs.new_version }}
+          next_release_version=${next_release_version%-dev}
```

Comment:
Should there really be an `-dev` ending as well as a `(development release)`?

Reply:
Ah, not quite! The `${var%-dev}` syntax actually _removes_ the `-dev` suffix from `${var}`.

See https://www.gnu.org/software/bash/manual/html_node/Shell-Parameter-Expansion.html for more details.